packages/solana-wallet-standard/src/register.ts

```diff
@@ -0,0 +1,69 @@
+// This is copied from @wallet-standard/wallet
```

Comment:
Is it not possible to import these from that package?

Reply:
don't know, followed the guide here: https://github.com/solana-labs/wallet-standard/blob/master/WALLET.md

Reply:
Wow ok, they got a very rough approach for this at the moment it seems

Reply:
didn't realise it was a clone